src/main/java/io/jenkins/plugins/coverage/CoverageProcessor.java

```diff
@@ -107,6 +118,7 @@ public CoverageResult processCoverageReport() throws IOException, InterruptedExc
 
         // If enable automatically detecting, it will try to find report and correspond adapter.
         if (isEnableAutoDetect()) {
+            listener.getLogger().println("Auto Detect is enabled: Looking for reports...");
```

Comment:
These end up going to the build log - I wonder if we should use java logging instead?

Reply:
Might be something for discussion, but even having a logger for this might not be useful since the java logging most likely gets flooded with other logs (would be difficult to find if debugging something). Maybe having a "debug" option 

For now I'd recommend putting `listener.getLogger()` into a variable and removing duplicate method calls everywhere else which would help with future improvements.

Reply:
You can set up different types of loggers in Jenkins though, so it's possible to configure it to see only jogs from this plugin.

Reply:
e.g. https://wiki.jenkins.io/display/JENKINS/Logging